src/pages/index.js

```diff
@@ -1,38 +1,42 @@
-import "./index.css";
-import { enableValidation, settings } from "../scripts/validation";
+import "../pages/index.css";
+import {
+  enableValidation,
+  settings,
+  resetValidation,
+} from "../scripts/validation.js";
```

Comment:
Please, import the `disableButton()` function from the `validation.js` file, to be able to disable the save buttons in `.then` blocks


```js

import {
  enableValidation,
  settings,
  resetValidation,
  disableButton,
} from "../scripts/validation.js";
```


But don't forget to add `export` for the function inside `validation.js`


```js

export const disableButton = (buttonElement, config) => {

```